girder_volview/dicom.py

```diff
@@ -0,0 +1,120 @@
+import datetime
+
+import pydicom
+import pydicom.valuerep
+import pydicom.multival
+import pydicom.sequence
+
+from girder.models.item import Item
+from girder.models.file import File
+
+
+# Code modified from https://github.com/girder/girder/blob/master/plugins/dicom_viewer/girder_dicom_viewer/__init__.py
+def addDicomTagsToItem(event):
+    """
+    Add DICOM tags to Item metadata.
+    """
+    file = event.info["file"]
+    dicomMetadata = _parseFile(file)
+    if dicomMetadata is None:
+        return
+
+    itemMeta = {"dicom": dicomMetadata}
+    item = Item().load(file["itemId"], force=True)
+    Item().setMetadata(item, itemMeta)
```

Comment:
I ditched `girder_dicom_viewer`s cleaning of Item metadata non maching DICOM tags when addtional files are added to an Item. But could bring it back if helpful.